### DISCUSSION TITLE
Running renovate from Centralized repo

### DISCUSSION THREAD

**Discussion by rajat2911** [renovatebot/renovate#29052] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Hi Team, 
I have around 10 repos where we have dependencies for helm, docker, go lang , terraform etc . 
My existing setup is like I have created a centralised repository named as Renovate and running a GitHub action which periodically runs on every repo and created a PR if any dependency updates are found where I need to setup 10 renovate.json individually for each repo . 
The question is can it possible to have all the renovate.json config file at one central location and renovate when run for a repo then fetches its configuration file from the central repo . This will save time for me to if I want to do changes in the renovate config then I need to do changes at multiple locations?

Thanks in advance 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
yes, shared presets are the recommended way, so a repo can override if required 

https://docs.renovatebot.com/config-presets/
